clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/config/AuditLoggingConfiguration.java

```diff
@@ -46,11 +46,8 @@ public class AuditLoggingConfiguration {
   @JsonProperty("security_context")
   private SecurityContext securityContext;
 
-  @JsonProperty(value = "jvs_endpoint")
-  private String jvsEndpoint = "localhost:8080";
-
-  @JsonProperty(value = "require_justification")
-  private boolean justificationRequired = false;
+  @JsonProperty("justification")
+  private Justification justification = new Justification();
```

Comment:
I'm guessing that if this were to default to null, `auditLoggingConfiguration.getJustification().isEnabled()` would throw null pointer exception, so this seems good to me.

Reply:
Looks like for `SecurityContext`, `ConditionConfig` they use the null as the default value. And here is the similar discussion in Golang: https://github.com/abcxyz/lumberjack/pull/284#discussion_r960092449

Reply:
Yes, since Go client set it to null, will make the change for consistancy.

Reply:
At first, I didn't catch the point that Mete mentioned that `auditLoggingConfiguration.getJustification().isEnabled() would throw null pointer exception`. In go client, when we use `justification`, to avoid the null pointer exception, we will check whether `justisfication` is nil or not.  https://github.com/abcxyz/lumberjack/blob/563a9dd75e25d3645e6bebecc506d7770679f17c/clients/go/pkg/auditopt/config.go#L173.